src/cache/intern.rs

```diff
@@ -40,6 +40,14 @@ cfg_if! {
                 }
             })
         }
+
+        fn unintern_str(key: &str) {
```

Comment:
Should we have some sort of reference counter perhaps? That is, if several parts of code use `intern`, but then one does `unintern`, maybe we shouldn't evict it yet?

Reply:
I figured that if you want that, it's easy enough to do your own ref-counting in your program.

I'm a bit wary of introducing too much complexity, it seems to me that ref-counting is orthogonal to interning.